.github/workflows/sanityTesting.yml

```diff
@@ -0,0 +1,31 @@
+name: Run sanity tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          sudo apt-get install python-opengl libglfw3 python3-wheel -y
+          python -m pip install --upgrade pip
+          pip install wheel
+          pip install -r requirements.txt
+      - name: Set env
+        # TODO: Find an alternate way to get the path, instead of hard-coding
+        run: echo "PYTHONPATH=$PYTHONPATH:${HOME}/work/p5" >> $GITHUB_ENV
```

Comment:
I have hardcoded this path, but it will break if GitHub will change its configuration of running actions in a different folder inside the VM. We can keep this for now, but it is a potential bug.

Reply:
Similarly, I think `pip install .` could also work here.

Reply:
Worked perfectly. Thanks!